Formula/t/trzsz-ssh.rb

```diff
@@ -24,11 +24,11 @@ def install
   end
 
   test do
-    assert_match "trzsz ssh #{version}", shell_output("#{bin}/tssh -V")
-    assert_match "trzsz ssh #{version}", shell_output("#{bin}/tssh --version")
+    assert_match "trzsz ssh #{version}", shell_output("#{bin}/tssh -V 2>&1")
+    assert_match "trzsz ssh #{version}", shell_output("#{bin}/tssh --version 2>&1")
 
-    assert_match "invalid option", shell_output("#{bin}/tssh -o abc", 255)
-    assert_match "invalid bind specification", shell_output("#{bin}/tssh -D xyz", 255)
-    assert_match "invalid forward specification", shell_output("#{bin}/tssh -L 123", 255)
+    assert_match "invalid option", shell_output("#{bin}/tssh -o abc 2>&1", 255)
+    assert_match "invalid bind specification", shell_output("#{bin}/tssh -D xyz 2>&1", 255)
+    assert_match "invalid forward specification", shell_output("#{bin}/tssh -L 123 2>&1", 255)
```

Comment:
but the thing is why do you output version info into stderr?

Reply:
The original intention was to output the help message to stderr, but it affected the output of version.

https://github.com/trzsz/trzsz-ssh/blob/92d1967bdb24c0cab26428b49ed502c08d8bb9af/tssh/main.go#L122-L128

Reply:
can we not output the version or help to the stderr? I dont think that is right practice?

Reply:
But tssh v0.1.22 has been released and cannot be changed now.

Reply:
(also you need to squash the commits to get rid of `autosquash` label and maybe good to add a note there too)

Reply:
> can you log an issue and address in the future? thanks!

https://github.com/trzsz/trzsz-ssh/issues/136